test/acceptance/features/workloadResourceMapping.feature

```diff
@@ -0,0 +1,643 @@
+                  - version: "*"
+                    volumes: .spec.volumeData
+                    containers:
+                      - path: .spec.containerSpecs[*]
+                        name: .id
+                        env: .envData
+                        volumeMounts: .volumeEntries
+                      - path: .spec.initContainerSpecs[*]
+                        name: .id
+                        env: .envData
+                        volumeMounts: .volumeEntries
+            """
+
+    @spec
+    Scenario: Deployment identity mappings should not change binding behavior
+        Given Generic test application is running
+        And The workload resource mapping is present
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: deployments.apps
+            spec:
+                versions:
+                  - version: "*"
+                    containers:
+                      - path: .spec.template.spec.containers[*]
+                        name: .name
+                        env: .env
+                        volumeMounts: .volumeMounts
+                      - path: .spec.template.spec.initContainers[*]
+                        name: .name
+                        env: .env
+                        volumeMounts: .volumeMounts
+                    volumes: .spec.template.spec.volumes
+            """
+        And The custom resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: $scenario_id-backend
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/username: path={.spec.username}
+            spec:
+                host: example.common
+                username: foo
+            """
+        When Service Binding is applied
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id-binding
+            spec:
+                type: mysql
+                service:
+                    apiVersion: stable.example.com/v1
+                    kind: Backend
+                    name: $scenario_id-backend
+                workload:
+                    name: $scenario_id
+                    apiVersion: apps/v1
+                    kind: Deployment
+            """
+        Then Service Binding is ready
+        And The application env var "SERVICE_BINDING_ROOT" has value "/bindings"
+        And Content of file "/bindings/$scenario_id-binding/host" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/bindings/$scenario_id-binding/type" in application pod is
+            """
+            mysql
+            """
+        And The workload resource mapping is deleted
```

Comment:
The step should match the step definition - that is `The Workload Resource Mapping is deleted` (incl. all other occurencies)

Reply:
Fixed.